frontend/src/components/BusDetailsCard/index.tsx

```diff
@@ -19,34 +19,37 @@ function BusDetailsCard() {
   const time = useOrderStore(state => state.time);
   const source = useOrderStore(state => state.source);
   const destination = useOrderStore(state => state.destination);
+
   return (
     <>
       <Box
         sx={{
           display: 'flex',
-          alignItems: 'center',
-          gap: {xs: '10px', md: '2rem'},
+          flexDirection: {xs: 'row', sm: 'row'},
+          alignItems: {xs: 'center', sm: 'center'},
+          gap: {xs: '10px', sm: '2px', md: '10px'},
+          margin: {xs: '0', md: '0 0.5rem'},
         }}
       >
         <Box
           sx={{
             display: 'flex',
             alignItems: 'center',
-            gap: {xs: '5px', md: '10px'},
+            gap: {xs: '5px', md: '5px'},
```

Comment:
let's write it as `gap: '5px'` as both produce the same result, so we should opt the one having better readability.